apps/api/src/app/messages/e2e/remove-messages-by-transactionId.e2e.ts

```diff
@@ -71,11 +56,10 @@ describe('Delete Messages By TransactionId - /messages/?transactionId= (DELETE)'
 
     expect(messages.length).to.be.greaterThan(0);
 
-    await axiosInstance.delete(`${session.serverUrl}/v1/messages/transaction/${transactionId}`, {
-      headers: {
-        authorization: `ApiKey ${session.apiKey}`,
-      },
-    });
+    if (transactionId == null) {
+      throw new Error('must have transaction id');
```

Comment:
Should we throw an error at this point or expect that the transaction is not null using the e2e assertions?

Reply:
that's a good catch, the DTO was not configured correctly, this is not optional, it's always there, will be removed